src/Elasticsearch/ConnectionPool/Selectors/RoundRobinSelector.php

```diff
@@ -29,8 +29,10 @@ class RoundRobinSelector implements SelectorInterface
      */
     public function select($connections)
     {
+        $returnConnection = $connections[$this->current % count($connections)];
```

Comment:
This change means, that it would select the first connection (with the array index `0`). Before it selected the second connection (with the array index `1`)